### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by MichelleDiamond** [MurhafSousli/ngx-scrollbar#667] [Q&A]
The sidebar was working perfectly until we moved to a mono repo. We are using the latest versions of bootstrap and angular

On mac the scrollbar is not visible. I have tried using extreme sizing and colours and have set my machine to always show scrollbars.

On windows (which I am unable to test) there is no scroll functionality at all and no scrollbars. 

I have imported the following using example 2 demo into the component

`import { NgScrollbarModule } from 'ngx-scrollbar';
import { NgScrollbar, NgScrollbarExt, ScrollViewport } from 'ngx-scrollbar';`

I have the following enclosing tag around the html

`<ng-scrollbar class="sidebar" [ngClass]="{ 'expanded': isExpanded }" id="sideMenu" visibility="hover" track="true" thumbSize="8px">`

I need urgent assistence please because this is now affecting our production site

Thank you

**Comment by MurhafSousli** [comment]
@MichelleDiamond I cannot tell what is going wrong with the info you provided, you need to reproduce your issue or if you want private support you can contact me by email: murhafsousli@gmail.com to arrange a meeting
